test/unit-test.js

```diff
@@ -0,0 +1,46 @@
+var expect = require('chai').expect;
+var assert = require('chai').assert;
+var should = require('chai').should;
```

Comment:
How come we're acquiring all three and why are we using `var` if we use `const` the rest of the time?